src/AiSystem.h

```diff
@@ -16,16 +16,26 @@ class AiSystem : public EventSystem<AiSystem> {
     void handle_object_transform(const Event& e);
     void handle_new_it(const Event& e);
 
+    void choose_ai_state(int car);
+    void avoid_car(int car, int to_avoid);
+    void pursue_car(int car, int to_pursue);
+    void random_car(int car);
+    void cutoff_car(int car, int to_catch);
     void path_to(int car, const glm::vec3& point_);
 
     int num_ai_;
 
+    enum class AiState
+    {Avoid, Pursue, Random, Cutoff};
```

Comment:
Enum values are constants and so would be nice  to be CAP_CASE

Also the `Avoid` and `Random`  are not being used right now, I think they should be used like the `Pursue` and `Cutoff` states.